examples/with_winit/src/lib.rs

```diff
@@ -376,7 +376,7 @@ fn run(
                 height,
                 antialiasing_method,
             };
-            let mut builder = SceneBuilder::for_scene(&mut scene);
+            scene.reset();
```

Comment:
I like that this is explicit now (instead of implicitly resetting the scene by creating a new fragment.

Reply:
This was a definite pain point and we have the complaints to prove it :)